orbit/changes/27275-windows-arm64

```diff
@@ -0,0 +1 @@
+- Added support for Windows ARM64 platform
```

Comment:
Would also add a comment here on how to generate the fleetd package (`fleetctl package --type=msi --arch=arm64 [...]`)